appengine-standard-java8/translate-pubsub/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>true</threadsafe>
+  <runtime>java8</runtime>
+  <application>YOUR-PROJECT-ID</application>
```

Comment:
This isn't always required, it can be set w/ mvn command line as well.

Reply:
I don't see any goal for setting project ID or other environment variables in any of the mvn commands. And I always found short build/run commands to be more helpful when developing so I could type them out instead of having to copy them from somewhere (if they had long incomprehensible vars instead of just being "mvn appengine:run," which is easily remembered).

Reply:
If the tag isn't present, it will grab from `gcloud` command.  `gcloud config list`.
Also, you can look at:
https://cloud.google.com/appengine/docs/standard/java/tools/maven-reference
`app.deploy.project`

Reply:
Ah OK, interesting. I found I had to use that tag when I was trying to run it because it was not finding any project ID, but maybe I was missing something in gcloud at the time.